### ISSUE TITLE
CUDA_ERROR_ILLEGAL_ADDRESS on GTX 1080

### ISSUE THREAD

**Issue by psnegi** [tensorflow/tensorflow#3422]
On training a model on cifar 10 I am getting following error after couple of epochs.

 tensorflow/stream_executor/cuda/cuda_driver.cc:1140] could not synchronize on CUDA context: CUDA_ERROR_ILLEGAL_ADDRESS :: No stack trace availablech: 034 | loss: 0.21259 - acc: 0.9223 -- iter: 08128/50000
F tensorflow/core/common_runtime/gpu/gpu_util.cc:370] GPU sync failed
Aborted (core dumped)

### Environment info
Operating System:
Ubuntu 14.04
4.2.0-27-generic #32~14.04.1-Ubuntu SMP Fri Jan 22 15:32:26 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

Installed version of CUDA and cuDNN: 
(please attach the output of `ls -l /path/to/cuda/lib/libcud*`):
-rw-r--r-- 1 root root   560184 Jul 18 17:20 /usr/local/cuda/lib64/libcudadevrt.a
lrwxrwxrwx 1 root root       16 Jul 18 17:20 /usr/local/cuda/lib64/libcudart.so -> libcudart.so.8.0
lrwxrwxrwx 1 root root       19 Jul 18 17:20 /usr/local/cuda/lib64/libcudart.so.8.0 -> libcudart.so.8.0.27
-rwxr-xr-x 1 root root   394472 Jul 18 17:20 /usr/local/cuda/lib64/libcudart.so.8.0.27
-rw-r--r-- 1 root root   737516 Jul 18 17:20 /usr/local/cuda/lib64/libcudart_static.a
-rwxr-xr-x 1 root root 78065952 Jul 18 18:03 /usr/local/cuda/lib64/libcudnn.so
-rwxr-xr-x 1 root root 78065952 Jul 18 18:03 /usr/local/cuda/lib64/libcudnn.so.5
-rwxr-xr-x 1 root root 78065952 Jul 18 18:03 /usr/local/cuda/lib64/libcudnn.so.5.0.5
-rw-r--r-- 1 root root 68709594 Jul 18 18:03 /usr/local/cuda/lib64/libcudnn_static.a

If installed from sources, provide the commit hash:
commit a3f61c1d5c76339e6c9655dac426bb3822659772

### Steps to reproduce
1. I have a model to train on cifar10,  which mainly consists of convolutions, PReLU and data augmentation.andaround r epoch  34 it get the above error. 
I have similar model running  simultaneously on  same GPU which run fine. Each of the model roughly
use 50% of GTX 1080 memory(TOTAL 8GB)
 
### What have you tried?
1. I tried to to run my model twice and getting same error.

**Comment by concretevitamin** [comment]
Could you check two things:

1. Try with CUDA 7.5, which is best supported.
2. Try with a smaller model footprint.  2 * ~50% seems to imply difficulty from memory pressure: we'd need to eliminate this from the root causes.

**Comment by aselle** [comment]
Automatically closing due to lack of recent activity. Please reopen if further information becomes available.
